Fix IceT build error:
.../OpenMPI/5.0.3-GCC-13.3.0/include/mpi_portable_platform.h:294:30: error: operator '/' has no right operand
  294 | #elif defined(__NVCOMPILER) // Must occur prior to PGI and CLANG
      |                              ^

Author: Samuel Moors (Vrije Universiteit Brussel)

--- ThirdParty/IceT/vtkicet/CMakeLists.txt.orig	2025-09-03 11:37:20.605848000 +0200
+++ ThirdParty/IceT/vtkicet/CMakeLists.txt	2025-09-03 12:16:42.405565759 +0200
@@ -175,8 +175,8 @@
 ENDIF()
 
 IF (CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_CLANG)
-  SET(ICET_C_FLAGS_WARN "-ansi -Wall -Wno-long-long -Wextra -Wformat-security -Wshadow -Wunused -Wreturn-type -Wpointer-arith -Wdeclaration-after-statement")
-  SET(ICET_C_FLAGS_WARN_LIST -ansi -Wall -Wno-long-long -Wextra -Wformat-security -Wshadow -Wunused -Wreturn-type -Wpointer-arith -Wdeclaration-after-statement)
+  SET(ICET_C_FLAGS_WARN "-std=c99 -Wall -Wno-long-long -Wextra -Wformat-security -Wshadow -Wunused -Wreturn-type -Wpointer-arith -Wdeclaration-after-statement")
+  SET(ICET_C_FLAGS_WARN_LIST -std=c99 -Wall -Wno-long-long -Wextra -Wformat-security -Wshadow -Wunused -Wreturn-type -Wpointer-arith -Wdeclaration-after-statement)
 ENDIF (CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_CLANG)
 
 # Configure testing support.
